src/Resources/skeleton/registration/RegistrationController.tpl.php

```diff
@@ -35,11 +54,23 @@ public function register(Request $request, UserPasswordEncoderInterface $passwor
                     $form->get('plainPassword')->getData()
                 )
             );
-
+<?php if (!$will_verify_email): ?>
             $entityManager = $this->getDoctrine()->getManager();
             $entityManager->persist($user);
             $entityManager->flush();
+<?php else: ?>
+            $this->entityManager->persist($user);
+            $this->entityManager->flush();
```

Comment:
It's a little screwy that:

A) without confirmation, we use `$entityManager`
B) with confirmation, we use `$this->entityManager`

Probably we should just use the `AbstractController` shortcut everwhere: `$entityManager = $this->getDoctrine()->getManager();` and then use the local variable (and then no constructor arg).

Reply:
done - this was an artifact from the trait that we no longer use..